### PR TITLE
Fix GPG keys for Rocky 8 (and perhaps also other RedHat-bases systems)

### PR DESCRIPTION
**Pull request by gdubicki**
This is the solution by @mkondratev, copied from https://github.com/geerlingguy/ansible-role-rabbitmq/pull/14.

It fixes the error `Failed to validate GPG signature for rabbitmq-server-3.8.16-1.el8.noarch` on my Rocky 8 machine.

### CODE REVIEW

**Comment by gdubicki** [comment]
Hm, it seems that this has also been done in https://github.com/geerlingguy/ansible-role-rabbitmq/pull/8 but it did not get merged...

**Comment by gdubicki** [comment]
This is not stale, it's still valid.

**Comment by gdubicki** [comment]
Ugh, not stale.

File: tasks/setup-RedHat.yml [hunk 1]
```diff
@@ -4,10 +4,15 @@
...
+- name: Add GPG keys.
   rpm_key:
-    key: "https://packagecloud.io/gpg.key"
+    key: "{{ item }}"
     state: present
+  with_items:
+    - "https://packagecloud.io/gpg.key"
+    - "https://packagecloud.io/rabbitmq/erlang/gpgkey"
+    - "https://packagecloud.io/rabbitmq/rabbitmq-server/gpgkey"
+    - "https://github.com/rabbitmq/signing-keys/releases/download/2.0/rabbitmq-release-signing-key.asc"
```
Comment (geerlingguy) [inline tasks/setup-RedHat.yml:15]: Is this relevant for future releases of rabbitmq as well, though?

Comment (gdubicki) [reply tasks/setup-RedHat.yml:15]: Sorry, I don't understand the question. Are you referring to some announcement about changes planned for future RabbitMQ releases?

The keys listed here match the ones that RabbitMQ currently recommends, see https://www.rabbitmq.com/install-rpm.html#package-cloud.

**Pull request merged**